src/iframe/src/Iframe.js

```diff
@@ -839,6 +852,7 @@ class Iframe extends Component {
                   selectedActors.includes(d.name)
                 )
               })
+              this.drawPixels()
```

Comment:
Sorry for my ignorance - why is this call needed?

Reply:
Its not. My bad. I added it while trying to fix drawActors, but this will get fixed later.